### ISSUE TITLE
global_step always equals to zero

### ISSUE THREAD

**Issue by MYWmiss** [sogou/SMRCToolkit#24]

**Comment by MYWmiss** [comment]
tensorboard
the point of loss float up and down the Y axis

**Comment by yylun** [comment]
@MYWmiss yes, we miss the global_step parameter in defining train_op.

To make global_step increase, take BiDAF as example, change https://github.com/sogou/SMRCToolkit/blob/master/sogou_mrc/model/bidaf.py#L214-L216 to
```python
def compile(self, optimizer, initial_lr):
    self.optimizer = optimizer(initial_lr)
    global_step = tf.train.get_or_create_global_step()
    self.train_op = self.optimizer.minimize(self.loss, global_step=gloal_step)
```

Thx for pointing it out, will fix soon 😃
